libvalkeycluster/valkeycluster.c

```diff
@@ -1529,70 +1533,70 @@ void redisClusterFree(redisClusterContext *cc) {
     }
 
     if (cc->username != NULL) {
-        hi_free(cc->username);
+        vk_free(cc->username);
         cc->username = NULL;
     }
 
     if (cc->password != NULL) {
-        hi_free(cc->password);
+        vk_free(cc->password);
         cc->password = NULL;
     }
 
-    hi_free(cc);
+    vk_free(cc);
 }
 
-/* Connect to a Redis cluster. On error the field error in the returned
+/* Connect to a Valkey cluster. On error the field error in the returned
  * context will be set to the return value of the error function.
  * When no set of reply functions is given, the default set will be used. */
-static int _redisClusterConnect2(redisClusterContext *cc) {
+static int _valkeyClusterConnect2(valkeyClusterContext *cc) {
 
     if (cc->nodes == NULL || dictSize(cc->nodes) == 0) {
-        __redisClusterSetError(cc, REDIS_ERR_OTHER,
-                               "servers address does not set up");
-        return REDIS_ERR;
+        __valkeyClusterSetError(cc, VALKEY_ERR_OTHER,
+                                "servers address does not set up");
+        return VALKEY_ERR;
     }
 
-    return redisClusterUpdateSlotmap(cc);
+    return valkeyClusterUpdateSlotmap(cc);
 }
 
-/* Connect to a Redis cluster. On error the field error in the returned
+/* Connect to a Valkey cluster. On error the field error in the returned
  * context will be set to the return value of the error function.
  * When no set of reply functions is given, the default set will be used. */
-static redisClusterContext *_redisClusterConnect(redisClusterContext *cc,
-                                                 const char *addrs) {
+static valkeyClusterContext *_valkeyClusterConnect(valkeyClusterContext *cc,
```

Comment:
I think single underscore might also be problematic. Maybe `valkeyClusterConnect_impl` / `valkeyClusterConnectImpl`?

Reply:
Separate PR

Reply:
Added issue #19